apps/prairielearn/src/schemas/schemas/infoAssessment.json

```diff
@@ -324,6 +324,24 @@
         "advanceScorePerc": {
           "$ref": "#/definitions/advanceScorePerc"
         },
+        "canSubmit": {
+          "description": "A list of group role names that can submit questions in this zone. Only applicable for group assessments.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "uniqueItems": true,
+          "default": []
+        },
+        "canView": {
+          "description": "A list of group role names that can view questions in this zone. Only applicable for group assessments.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "uniqueItems": true,
+          "default": []
+        },
```

Comment:
Weren't these already added in a previous PR? Need to figure out why they show up here in the changes

Reply:
Seems like these didn't line up with the cherry picked commits from before, so they'll have to be removed. They're duplicate keys.